src/accelerate/utils/dataclasses.py

```diff
@@ -415,6 +415,16 @@ class ProjectConfiguration:
         metadata={"help": "The current save iteration."},
     )
 
+    save_on_each_node: bool = field(
+        default=False,
+        metadata={
+            "help": (
+                "When doing multi-node distributed training, whether to save models and checkpoints on each node, or"
+                " only on the main one"
```

Comment:
If this is something that can be left as default for every setting except for DeepSpeed, maybe it's worth mentioning here. If such a blanket statement cannot be made, ignore my comment.